### ISSUE TITLE
Select sounds that turns off by itself and won't automatically turn it back on fix after going in the app that it's launching

### ISSUE THREAD

**Issue by TheLegendaryKillerXD** [yuliskov/SmartYouTubeTV#194]
I'm having a lot of problems with them on Amazon Fire TV stick 2nd generation. It keeps on turning itself off and back on when it always does. And the select sound while scrolling that I didn't hear is that it didn't stay automatically on by itself. Can you please give it a try and have a good fix, so that it can be done in no time?

**Comment by TheLegendaryKillerXD** [comment]
This needs to be done from right before to after. Like it won't be happening to get a lot of problems again. 
![20181123_205451](https://user-images.githubusercontent.com/45300240/48963910-17820d80-ef63-11e8-8d4d-f030c0550963.jpg) 
![20181123_205549](https://user-images.githubusercontent.com/45300240/48963912-1c46c180-ef63-11e8-960d-b1636447960b.jpg)

**Comment by yuliskov** [comment]
@TheLegendaryKillerXD you've said that sound setting not have effect or what?

**Comment by TheLegendaryKillerXD** [comment]
No, it doesn't. And it really happens to me in the first time from 1080p resolution for the main version. And this situation needs to be solved. So, it can get to normal that I can hear that I'm selecting something again.

**Comment by yuliskov** [comment]
@TheLegendaryKillerXD sorry, I still don't get it.

**Comment by TheLegendaryKillerXD** [comment]
I'm talking about the sound effects it needs to be fixed that has to get automatically turned on and not off anymore when the next new version stable release comes. Can you please see and check if you can test it out when it's gonna be happening?

**Comment by TheLegendaryKillerXD** [comment]
Thank you!
